github/github_test.go

```diff
@@ -1136,6 +1141,36 @@ func TestDo_rateLimit(t *testing.T) {
 	}
 }
 
+func TestDo_rateLimitCategory(t *testing.T) {
+	if c := category(http.MethodGet, "/"); c != coreCategory {
```

Comment:
In reason that this is not a table-driven test?

I'm OK leaving it as-is if you want... since it would probably be the same number of lines, but I often find that table-driven tests are easier to read because you can just look at the data all in one place.

But no biggie... I'm fine either way.

Reply:
tbh I just tried to follow TestRateLimits & TestRateLimits_overQuota,
but I fixed it to be table-driven (and fixed those two while at it)